### ISSUE TITLE
Bioghist notes associated with agent records should display with agents and in finding aids

### ISSUE THREAD

**Issue by faithc** [pulibrary/pulfalight#584]
Biographical/historical notes that are now associated with agent records should display as part of the name records AND display as notes in finding aids in the Collection Description tab/page and at the component level with which agent records are associated a la PULFA 2.0 (examples: https://findingaids.princeton.edu/collections/C1599#description + https://findingaids.princeton.edu/names/wh247083m; https://findingaids.princeton.edu/names/2j62sc10p + https://findingaids.princeton.edu/collections/C0391/c1501). 
 
Whereas the [Collection-level notes spreadsheet](https://docs.google.com/spreadsheets/d/1fKd5t9Xk_Ph1MD5zt2O2LwSWfLeNQjBYtDM6fk8qH9k/edit#gid=0) indicates that bioghist notes are located in EADs, most are now only associated with/part of agent records. Note, there will still be some bioghist notes in Resource records.

**Comment by tpendragon** [comment]
Blocked by https://github.com/pulibrary/princeton_ead_exporter/issues/4

**Comment by faithc** [comment]
Sorry, @tpendragon, why is this blocked? Does this have to do with waiting for ASpace data?

**Comment by faithc** [comment]
Might this also be doable for pulfa 2.0 while it's still in use?

**Comment by tpendragon** [comment]
It's just blocked because it has a dependency. We have to update our aspace plugin to embed agent bioghists at the component level. We're not waiting on anyone, there's just another ticket to do first.

**Comment by faithc** [comment]
Gotcha. Thanks for explaining. :)

**Comment by tpendragon** [comment]
The feature's been added to our plugin, we're just waiting on Lyrasis to update our plugin.
